src/auto_scheduler/compute_dag.cc

```diff
@@ -114,7 +118,432 @@ Array<te::Operation> TopoSortOps(const Array<te::Tensor>& tensors) {
   return ops;
 }
 
-// Estimate number of float operations in an expression
+// Extract all tensor accesses in an expr
+class TensorAccessExtractor : public StmtExprVisitor {
```

Comment:
A second look: looks like it is only applied to PrimExpr
```suggestion
class ReadFromAndHasBranchExtractor : public ExprVisitor {
```